pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>org.graylog.plugins</groupId>
     <artifactId>graylog-plugin-dnsresolver</artifactId>
-    <version>1.2.0-SNAPSHOT</version>
+    <version>1.1.3-nt1</version>
```

Comment:
Please remove the custom version and use the old version string.

Reply:
Also please update the `graylog2.version` property.

The changed plugin will not run with Graylog 2.0.0.

Reply:
So I will update the graylog2.version to 2.4.0, so release 1.2.x will target graylog 2.4+ then.